data/transactions/logic/eval.go

```diff
@@ -24,6 +24,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/algorand/go-algorand/crypto"
```

Comment:
group this with the other similar imports

Reply:
fixed.